account/src/main/java/com/caldevsupplychain/account/controller/AccountController.java

```diff
@@ -0,0 +1,263 @@
+package com.caldevsupplychain.account.controller;
+
+
+import com.caldevsupplychain.account.dto.UserEditDTO;
+import com.caldevsupplychain.account.dto.UserLoginDTO;
+import com.caldevsupplychain.account.dto.UserSignupDTO;
+import com.caldevsupplychain.account.model.User;
+import com.caldevsupplychain.account.service.AccountService;
+import com.caldevsupplychain.account.util.AccountWSBuilder;
+import com.caldevsupplychain.account.validator.EditUserValidator;
+import com.caldevsupplychain.account.validator.LoginValidator;
+import com.caldevsupplychain.account.validator.SignupValidator;
+import com.caldevsupplychain.common.config.WebConfig;
+import com.caldevsupplychain.common.type.ErrorCode;
+import com.caldevsupplychain.common.ws.account.ApiErrorsWS;
+import com.caldevsupplychain.common.ws.account.UserWS;
+import com.caldevsupplychain.notification.mail.service.EmailService;
+import io.swagger.annotations.Api;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.shiro.SecurityUtils;
+import org.apache.shiro.authc.AuthenticationException;
+import org.apache.shiro.authc.UsernamePasswordToken;
+import org.apache.shiro.subject.Subject;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.context.request.WebRequest;
+
+import javax.mail.MessagingException;
+import javax.servlet.Servlet;
+import javax.servlet.ServletRequest;
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.constraints.Null;
+import javax.websocket.server.PathParam;
+import java.util.Locale;
+import java.util.Optional;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/account/v1")
+@AllArgsConstructor
+public class AccountController {
+
+    private ModelMapper modelMapper;
```

Comment:
Remove all the private fields after changing this class to an interface

Reply:
fixed